.prettierrc.json

```diff
@@ -3,7 +3,8 @@
     {
       "files": "*.sol",
       "options": {
-        "printWidth": 100
+        "printWidth": 100,
```

Comment:
in ma3 we put 120, what should we do?

Reply:
I will put the same everywhere to standardize the different repo.